src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  if (numberOne !== null && numberTwo !== null && operation !== null) {
+    const num1 = Big(numberOne);
+    const num2 = Big(numberTwo);
+    if (operation === '+') {
+      return num1.plus(num2);
+    } if (operation === '-') {
+      return num1.minus(num2);
+    } if (operation === 'X') {
+      return num1.times(num2);
+    } if (operation === '÷') {
+      if (num2 !== 0) {
+        return num1.div(num2);
+      }
+      return 'Infinite';
```

Comment:
Is there a way to handle operations on value that can't be converted to numbers?

I mean after `Infinite` is displayed on the screen, what happen if a user adds `2` to `infinite`? :thinking:

Reply:
I will check that in detail. Thank you!